commands/scanpullrequests.go

```diff
@@ -0,0 +1,137 @@
+package commands
+
+import (
+	"context"
+	"sort"
+	"strings"
+
+	"github.com/jfrog/frogbot/commands/utils"
+	"github.com/jfrog/froggit-go/vcsclient"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	clitool "github.com/urfave/cli/v2"
+)
+
+func ScanPullRequests(c *clitool.Context) error {
+	// Get params and VCS client
+	params, client, err := utils.GetParamsAndClient()
+	if err != nil {
+		return err
+	}
+	// Send usage report
```

Comment:
I suggested to @sverdlov93 in his PR to make changes to the `Send usage report` flow for all commands, so best to stay in sync with him about this.